src/database/comment.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from 'mongoose';
+import IComments from '../interfaces/comment';
+
+const commentsSchema = new mongoose.Schema<IComments>({
+  commentBody: {
+    type: String,
+  },
+  user: {
+    type: String,
```

Comment:
change the type here from string to mongoose object id>>> mongoose.Schema.Types.ObjectId